tracegen/internal/tracegen/config.go

```diff
@@ -54,6 +73,11 @@ func (c *Config) Flags(fs *flag.FlagSet) {
 	fs.StringVar(&c.Endpoint, "otlp-endpoint", "localhost:4317", "Target to which the exporter is going to send spans or metrics. This MAY be configured to include a path (e.g. example.com/v1/traces)")
 	fs.BoolVar(&c.Insecure, "otlp-insecure", false, "Whether to enable client transport security for the exporter's grpc or http connection")
 	fs.BoolVar(&c.UseHTTP, "otlp-http", false, "Whether to use HTTP exporter rather than a gRPC one")
+
+	// custom headers
+	c.Headers = make(map[string]string)
+	fs.Var(&c.Headers, "otlp-headers", "Custom headers to be passed along with each OTLP request. The value expected in the format key=value."+
```

Comment:
Either rename here to `otlp-header` (singular) or change the help text to be `otlp-headers` (plural). One of them is wrong :-)

Reply:
Done. `-otlp-header` it is.

Reply:
`./tracegen -otlp-http -otlp-header a=b -otlp-header c=dk=  -otlp-insecure -otlp-endpoint localhost:9000 -traces 1`